package.json

```diff
@@ -14,7 +14,7 @@
     "styled-components": "^4.3.2"
   },
   "scripts": {
-    "start": "REACT_APP_API_URL=http://localhost:9000 craco start",
+    "start": "craco start",
```

Comment:
❓ I'm guessing removing this is the temporary solution to this [bug](https://trello.com/c/ojwmeY2e/73-yarn-start-not-recognizing-env-variable), right?

We should talk about approaching this. Will start a discussion on it

Reply:
This was a mistake because of the merge mess that we had haha, let me revert it